pkg/plugins/mellanox/mellanox_plugin.go

```diff
@@ -19,6 +19,7 @@ type MellanoxPlugin struct {
 	helpers     helper.HostHelpersInterface
 }
 
+var pciAddressesToChange []string
```

Comment:
I'd prefer to have this in the `MellanoxPlugin` struct, but I decided to create a global var to keep the style.
I will think about refactoring the mellanox plugin code

Reply:
Shouldn't this be closer to pciAddressesToReset?

Reply:
You are right, will name it like this